tests/tools/output.py

```diff
@@ -6,8 +6,10 @@ def print_results(results):
         else:
             print("ERROR | " + i[0])
 
-def print_result(name, result):
+def print_result(name, function):
+    print(name + "... ", end='', flush=True)
+    result = function()
     if result is True:
-        print("OK | " + name)
+        print("\t\33[32m OK \33[0m")
```

Comment:
```suggestion
        print(" \33[32m OK \33[0m")
```

Reply:
done, didn't add space as it's already after the color change